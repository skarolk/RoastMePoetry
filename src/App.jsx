import React from "react";
import logo from "./logo.svg";
import "./App.css";
import NavMenu from "./Header/NavMenu/NavMenu";
import Button from "react-bootstrap/Button";
import "bootstrap/dist/css/bootstrap.min.css";

//@ts-ignore
//tslint:disable-next-line
import BoilerplateComponent from 'jw-react-npm-boilerplate';

function App() {
  return (
    <>
      <NavMenu />
      <div className="App">
        <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <p>When you hate enough to send the very worst</p>
          <Button variant="outline-info">Buy Now</Button>
          <BoilerplateComponent label="My React boilerplate component label: " onChange={val => console.log(val)} />
        </header>
      </div>
    </>
  );
}

export default App;
