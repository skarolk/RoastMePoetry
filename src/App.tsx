import React from "react";
import logo from "./logo.svg";
import "./App.css";
import NavMenu from "./Header/NavMenu/NavMenu";
import Button from "react-bootstrap/Button";
import "bootstrap/dist/css/bootstrap.min.css";

function App() {
  return (
    <>
      <NavMenu />
      <div className="App">
        <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <p>When you hate enough to send the very worst</p>
          <Button variant="outline-info">Buy Now</Button>
        </header>
      </div>
    </>
  );
}

export default App;
