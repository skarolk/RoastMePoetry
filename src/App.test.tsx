import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import App from "./App";

test("renders correctly", () => {
  const RenderedApp = renderer.create(<App />).toJSON();
  expect(RenderedApp).toMatchSnapshot();
});

test("renders learn react link", () => {
  const { getByText } = render(<App />);
  const linkElement = getByText("When you hate enough to send the very worst");
  expect(linkElement).toBeInTheDocument();
});
