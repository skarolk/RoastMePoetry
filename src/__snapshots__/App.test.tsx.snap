// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <nav
    className="navbar navbar-expand navbar-dark bg-dark sticky-top"
  >
    <a
      className="navbar-brand"
      href="#home"
    >
      <img
        alt=""
        className="d-inline-block align-top"
        height="30"
        src="https://react-bootstrap.github.io/logo.svg"
        width="30"
      />
       
      RoastMePoetry
    </a>
    <div
      className="justify-content-end navbar-nav"
      onKeyDown={[Function]}
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <button
        className="btn btn-outline-info"
        disabled={false}
        type="button"
      >
        Buy Now
      </button>
    </div>
  </nav>,
  <div
    className="App"
  >
    <header
      className="App-header"
    >
      <img
        alt="logo"
        className="App-logo"
        src="logo.svg"
      />
      <p>
        When you hate enough to send the very worst
      </p>
      <button
        className="btn btn-outline-info"
        disabled={false}
        type="button"
      >
        Buy Now
      </button>
    </header>
  </div>,
]
`;
